packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -11,7 +11,7 @@ const config = require('../../../config').getProperties();
 
 const mjmlConfig: Record<any, any> = {
   validationLevel: 'strict',
-  // filePath: path.join(__dirname, 'css', 'global.css'),
+  ignoreIncludes: config.env === 'test' ? true : false,
```

Comment:
I think this can just be `config.env === 'test'` since that evaluates to `true`/`false`.